corehq/apps/hqwebapp/static/hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.js

```diff
@@ -138,6 +138,18 @@ $.widget("ui.multiselect", {
 
 		$.widget.prototype.destroy.apply(this, arguments);
 	},
+	refresh: function() {
+		this.container.remove();
+		this._create();
+	},
+	addSelected: function(value, text) {
+		this.element.append(
+			$("<option/>").prop('value', value)
+			              .prop('selected', 'selected')
+			              .text(text)
+		);
+		this.refresh();
+	},
```

Comment:
It'd be nice to get a prettier multiselect widget.  This one had no way to add elements or refresh the UI via javascript, so I added that in.